fuji_server/helper/metadata_collector_rdf.py

```diff
@@ -438,9 +438,9 @@ def get_sparqled_metadata(self, g):
                 self.logger.info("FsF-F2-01M : Trying to query generic SPARQL on RDF, found triples: -:" + str(len(g)))
                 r = g.query(Mapper.GENERIC_SPARQL.value)
                 for row in r:
-                    for l, v in row.asdict().items():
-                        if l is not None:
-                            if l in [
+                    for link, v in row.asdict().items():
```

Comment:
Is `link` an appropriate name for `l`? Ruff did not like `l`, due to `ambiguous-variable-name`.

Reply:
I think its: v = related_resource; l = relation_type